CONTRIBUTING.md

```diff
@@ -1,103 +1,155 @@
-# Wandb library
-
-## Get the code / library
-
-Checkout from github:
+# Contributing to `wandb`
+
+We at Weights & Biases ❤️ open source and welcome contributions from the community!
+This guide discusses the development workflow and the internals of the `wandb` client library.
+
+### Table of Contents
+* [Development workflow](#development-workflow)
+* [Setting up your development environment](#setting-up-your-development-environment)
+* [Building protocol buffers](#building-protocol-buffers)
+* [Linting the code](#linting-the-code)
+* [Testing](#testing)
+  + [Overview](#overview)
+  + [Finding good test points](#finding-good-test-points)
+  + [Global Pytest Fixtures](#global-pytest-fixtures)
+  + [Code Coverage](#code-coverage)
+  + [Test parallelism](#test-parallelism)
+  + [Regression Testing](#regression-testing)
+* [Live development](#live-development)
+* [Code organization](#code-organization)
+* [Library Objectives](#library-objectives)
+* [Changes from production library](#changes-from-production-library)
+* [Detailed walk through of a simple program](#detailed-walk-through-of-a-simple-program)
+* [Documentation Generation](#documentation-generation)
+
+## Development workflow
+
+1. If you are a first-timer, please go to [`https://github.com/wandb/client`](https://github.com/wandb/client) 
+and click the "Fork" button in the top-right corner of the page. 
+This will create your personal copy of the repository that you will use for development. 
+   - Set up [SSH authentication with GitHub](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).
+   - Clone the forked project to your machine and add the upstream repository 
+     that will point to the main `wandb` project:
+    ```shell
+    git clone https://github.com/<your-username>/client.git
+    cd client
+    git remote add upstream https://github.com/wandb/client.git
+    ```
+   
+2. Browse the existing [Issues](https://github.com/wandb/client/issues) on GitHub to see
```

Comment:
Maybe this one should be the first point?

Reply:
👍🏻